### ISSUE TITLE
Incomplete sentence in WARC-Format 1.1 section 6.3.3 "response - other URI schemes"

### ISSUE THREAD

**Issue by sebastian-nagel** [iipc/warc-specifications#63]
In [section 6.3.3 ‘response’ - Other URI schemes](https://iipc.github.io/warc-specifications/specifications/warc-format/warc-1.1/#other-uri-schemes) the final sentence is truncated: "other URI schemes." This happened in 49250cc, ev. the second half of this sentence remained just by accident. However, the truncation has been observed when searching for a spec/recommendation how to calculate the checksum on payloads with "Transfer-Encoding: chunked" or any "Content-Encoding" except "identity". Any clarifications regarding the transfer encoding (cf. #22) have been removed here. Was this intended?

**Comment by JustAnotherArchivist** [comment]
Funny how nobody noticed that during the draft and for over two years since the release of WARC/1.1, and now we both discover it independently within two weeks. :-)

#61

**Comment by sebastian-nagel** [comment]
Yes, funny. Also overlaps with #62 as the statement regarding the marking of truncated responses has been removed in the same commit.

**Comment by ato** [comment]
I wouldn't be too surprised if this error was introduced when the markdown version was created. It doesn't seem to be present in the [PDF and Word drafts](http://bibnum.bnf.fr/warc/) hosted by BNF.

**Comment by JustAnotherArchivist** [comment]
Fixed by #79
